docs/how-xtrabackup-works.md

```diff
@@ -1,110 +1,101 @@
-commit.
-
-[Percona XtraBackup 8.0.30-23](release-notes/8.0/8.0.30-23.0.md) adds `--register-redo-log-consumer` parameter. The `--register-redo-log-consumer` parameter is disabled by default. When enabled, this parameter lets Percona XtraBackup register as a redo log consumer at the start of the backup. The server does not remove a redo log that Percona XtraBackup (the consumer) has not yet copied. The consumer reads the redo log and manually advances the log sequence number (LSN). The server blocks the writes during the process. Based on the redo log consumption, the server determines when it can purge the log.  
-
-*Percona XtraBackup* works by remembering the LSN when it starts, and then copying away the data files. It takes some time to do
-this, so if the files are changing, then they reflect the state of the database
-at different points in time. At the same time, *Percona XtraBackup* runs a
-background process that watches the transaction log files, and copies changes
-from it. *Percona XtraBackup* needs to do this continually because the
-transaction logs are written in a round-robin fashion, and can be reused after a
-while. *Percona XtraBackup* needs the transaction log records for every change
+Percona XtraBackup is based on how [`crash recovery`](https://dev.mysql.com/doc/refman/8.0/en/glossary.html#glos_crash_recovery) works. Percona XtraBackup copies the InnoDB data files, which results in data that is internally inconsistent; then performs crash recovery on the files to make a consistent, usable database again.
+
+This operation works because InnoDB maintains a redo log. This log contains a record of every change to the InnoDB data. At start, InnoDB inspects the data files and the redo log and performs the following
+steps:
+
+* Applies committed redo log entries to the data files
+* Performs an undo operation on any redo entries that modified data but did not commit
+
+## Version changes
+
+[Percona XtraBackup 8.0.30-23](release-notes/8.0/8.0.30-23.0.md) adds the `--register-redo-log-consumer` parameter, which is disabled by default. When enabled, this parameter lets Percona XtraBackup register as a redo log consumer at the start of the backup. The server does not remove a redo log that Percona XtraBackup (the consumer) has not yet copied. The consumer reads the redo log and manually advances the log sequence number (LSN). The server blocks the writes during the process. Based on the redo log consumption, the server determines when it can purge the log.  
+
+## Phases of XtraBackup
+
+The phases of an backup and restore process are the following:
+
+* Back up
+* Prepare
+* Restore
+
+
+## Back up
+
+Percona XtraBackup remembers the log sequence number (LSN) when it starts, and then copying away the data files. The backup takes time. If the data files change, the files reflect the database state at different points in time. The server writes the redo log in a circular fashion, and can be reused after a
```

Comment:
From 8.0.30, server no longer writes in a circular fashion. Instead, it creates new files and purges old redo log files.

Reply:
Done